packages/core/src/components/ColorPickerExperimental/components/ColorPickerItemComponent/ColorPickerItemComponent.tsx

```diff
@@ -0,0 +1,144 @@
+      const setHoverColor = (e: MouseEvent) => {
+        if (colorStyle === ColorStyle.SELECTED) {
+          (e.target as HTMLDivElement).style.background = ColorUtils.getMondayColorAsStyle(color, ColorStyle.REGULAR);
+        } else {
+          (e.target as HTMLDivElement).style.background = ColorUtils.getMondayColorAsStyle(color, ColorStyle.HOVER);
+        }
+      };
+      const restoreToOriginalColor = (e: MouseEvent) => {
+        (e.target as HTMLDivElement).style.background = colorAsStyle;
+      };
+      item.addEventListener("mouseenter", setHoverColor, false);
+      item.addEventListener("mouseleave", restoreToOriginalColor, false);
+
+      return () => {
+        item.removeEventListener("mouseenter", setHoverColor, false);
+        item.removeEventListener("mouseleave", restoreToOriginalColor, false);
+      };
+    }, [color, colorAsStyle, colorStyle, isMondayColor, itemRef, shouldRenderIndicatorWithoutBackground, isDisabled]);
+
+    const [isHovered, setIsHovered] = useState(false);
+
+    const handleMouseEnter = () => setIsHovered(true);
+    const handleMouseLeave = () => setIsHovered(false);
+
+    const shouldRenderIcon =
+      !isDisabled && ((isSelected && !isEditing) || ColorIndicatorIcon || (isEditing && isHovered));
+
+    const colorIndicatorWrapperStyle = shouldRenderIndicatorWithoutBackground ? { color: colorAsStyle } : {};
+    return (
+      <Tooltip content={tooltipContent}>
+        <li
+          className={cx(styles.itemWrapper, {
+            [styles.selectedColor]: isSelected && !isEditing,
+            [styles.active]: isActive,
+            [styles.circle]: colorShape === ColorShapes.CIRCLE,
+            [styles.disabled]: isDisabled
+          })}
+          data-testid={dataTestId || getTestId(ComponentDefaultTestId.COLOR_PICKER_ITEM, color)}
+          onMouseEnter={handleMouseEnter}
+          onMouseLeave={handleMouseLeave}
+        >
+          <div className={cx(styles.feedbackIndicator)} />
+          <Clickable
+            ref={itemRef}
+            ariaLabel={color}
+            className={cx(styles.colorItem, getStyle(styles, camelCase("color-item-size-" + colorSize)), {
+              [styles.colorItemTextMode]: shouldRenderIndicatorWithoutBackground
+            })}
+            style={{
+              background: isDisabled
+                ? disabledColor
+                : shouldRenderIndicatorWithoutBackground
+                ? "transparent"
+                : colorAsStyle
+            }}
+            onClick={onClick}
+            tabIndex="-1"
+            onMouseDown={e => e.preventDefault()} // this is for quill to not lose the selection
+            disabled={isDisabled}
+          >
+            <div className={cx(styles.colorIndicatorWrapper)} style={colorIndicatorWrapperStyle}>
+              {shouldRenderIcon && (
+                <Icon
+                  icon={isEditing ? Edit : isSelected ? SelectedIndicatorIcon : ColorIndicatorIcon}
```

Comment:
regarding the UX concern -
We did usability and all of the users started hovering over the colors and went to edit it. There is no issue in discoverability and understanding. It is the common and intuitive thing to do - hover over colors.
The second reason we didn't add it, is because it is not a consistent with other behaviors in the platform and in addition to that it is not a visually aesthetic solution